src/backend/distributed/transaction/lock_graph.c

```diff
@@ -124,9 +133,14 @@ BuildGlobalWaitGraph(void)
 	MultiConnection *connection = NULL;
 	foreach_ptr(connection, connectionList)
 	{
-		const char *command = "SELECT * FROM dump_local_wait_edges()";
+		StringInfo queryString = makeStringInfo();
+		const char *onlyDistributedTxStr = onlyDistributedTx ? "true" : "false";
 
-		int querySent = SendRemoteCommand(connection, command);
+		appendStringInfo(queryString,
+						 "SELECT * FROM dump_local_wait_edges(%s)",
```

Comment:
It's a pity we don't use an explicit column list here. We'll break rolling upgrades by returning additional columns.

I kind of wonder whether we should rename the function and keep the old signature.

Reply:
I ended up doing that